var/spack/repos/builtin/packages/mepo/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Mepo(PythonPackage):
+    """Tool to manage (m)ultiple git r(epo)sitories"""
+
+    homepage = "https://github.com/GEOS-ESM/mepo"
+    git = "https://github.com/GEOS-ESM/mepo.git"
+    pypi = "mepo/mepo-2.0.0rc3.tar.gz"
+
+    maintainers("pchakraborty", "mathomp4")
+
+    license("Apache-2.0", checked_by="mathomp4")
+
+    version("2.0.0rc3", sha256="c0c897a33f5018489e6cc14892961831c8922a3378ac30436496c52bf877aff7")
+
+    depends_on("python@3.9:", type=("build", "run"))
+    depends_on("py-colorama", type=("build", "run"))
+    depends_on("py-pyyaml@6:", type=("build", "run"))
+
```

Comment:
Is there a `[py-]mdutils` package missing?  Should there be a dependency on it?` 

```
dependencies = [
    "pyyaml>=6.0.1",
    "mdutils>=1.6.0",
]
```

Reply:
...and we realized `mdutils` is a development only requirement:
```toml
[tool.rye]
managed = true
dev-dependencies = [
    "black>=24.4.2",
    "pylint>=3.2.0",
    "flake8>=7.0.0",
    "pre-commit>=3.7.1",
    "mdutils>=1.6.0",
]
```

Reply:
I will work with @pchakraborty to get a new rc out tomorrow and move this PR to that.

Reply:
@mathomp4 I pushed a new version `2.0.0rc4` based on the latest `develop`.